cocos/editor-support/cocostudio/ActionTimeline/CCActionTimelineCache.cpp

```diff
@@ -882,6 +889,34 @@ Frame* ActionTimelineCache::loadInnerActionFrameWithFlatBuffers(const flatbuffer
     return frame;
 }
     
+Frame* ActionTimelineCache::loadBlendFrameWithFlatBuffers(const flatbuffers::BlendFrame* flatbuffers)
+{
+    BlendFuncFrame* frame = BlendFuncFrame::create();
+    cocos2d::BlendFunc blend;
+    blend.src = GL_ONE;
+    blend.dst = GL_ONE_MINUS_SRC_ALPHA;
+    if (nullptr != flatbuffers->blendFunc())
+    {
+        blend.src = flatbuffers->blendFunc()->src();
+        blend.dst = flatbuffers->blendFunc()->dst();
+    }
+    frame->setBlendFunc(blend);
+
+    int frameIndex = flatbuffers->frameIndex();
+    frame->setFrameIndex(frameIndex);
+
+    bool tween = flatbuffers->tween() != 0;
+    frame->setTween(tween);
+
+    auto easingData = flatbuffers->easingData();
```

Comment:
What does easing data do with blend frame ?
AFAIK, the blend frame cannot be animated, how does the easing function work ?